### PR TITLE
call window.URL.revokeObjectURL(file.preview) to avoid memory leak

### PR DESCRIPTION
**Pull request by stevemao**
fixes https://github.com/okonet/react-dropzone/issues/398

**What kind of change does this PR introduce?**

- [ ] bugfix
- [ ] feature
- [ ] refactoring / style
- [ ] build / chore
- [x] documentation

**Did you add tests for your changes?**

- [ ] Yes, my code is well tested
- [x] Not relevant

**If relevant, did you update the documentation?**

- [x] Yes, I've updated the documentation
- [ ] Not relevant

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,6 +99,8 @@ var DropzoneDemo = React.createClass({
 React.render(<DropzoneDemo />, document.body);
 ```
 
+*Important*: `react-dropzone` doesn't manage dropped files by itself. You need to destroyed the object URL yourself whenever you don't need to preview by calling `window.URL.revokeObjectURL(file.preview);` to avoid memory leak.
```
Comment (okonet) [inline README.md]: `destroyed` -> `destroy`

Comment (okonet) [inline README.md]: `leak` -> `leaks`

Comment (okonet) [inline README.md]: `don't need to preview` -> `don't need the preview`

**Comment by stevemao** [comment]
Fixed. Thanks :)

**Comment by okonet** [comment]
Awesome! Thanks for this

**Pull request merged**